tools/please_pex/BUILD

```diff
@@ -1,7 +1,40 @@
-# TODO(jpoole): This should be a remove file on the plugin repo's release page and we should remove //_please:please_pex
-filegroup(
+subinclude("//build_defs:python")
+
+go_binary(
+    name = "pex_main",
+    srcs = ["pex_main.go"],
+    deps = [
+        "//third_party/go:cli-init",
+        "//third_party/go:go-logging.v1",
+        "//tools/please_pex/pex",
+    ],
+)
+
+genrule(
     name = "please_pex",
-    srcs = ["@//_please:please_pex"],
-    labels = ["hlink:plz-out/package"],
+    srcs = [
+        ":pex_main",
+    ],
+    outs = ["please_pex"],
+    binary = True,
+    cmd = [
+        '"$TOOL" z -d -i . -o "$OUTS" --preamble_file $(location :pex_main)',
+    ],
+    tools = ["//tools:arcat"],
     visibility = ["PUBLIC"],
+    labels = ["hlink:plz-out/package"],
+)
```

Comment:
I don't think we need this rule anymore. We aren't zipping up the bootstrap deps to the pex tool anymore. I reckon the rule above and this one can be merge, without needing arcat?

Reply:
Good point!